modules/ROOT/pages/index.adoc

```diff
@@ -1,48 +1,96 @@
 
-* xref:ROOT:cassandra-data-migrator.adoc[{cass-migrator}]: It can be used in conjunction with the {product-proxy} for a migration with zero downtime. It can also be used on its own for migrations with acceptable downtime.
+  </div>
+  <div class="hidden lg:block flex basis-1/2 relative">
+++++
 
-* xref:ROOT:dsbulk-migrator.adoc[{dsbulk-migrator}]: {dsbulk-migrator} is an extension of {dsbulk-loader}.
-In addition to loading and unloading CSV and JSON data, {dsbulk-migrator} can transfer data between databases.
-It can read data from a table from your origin database and write it to a table in your target database.
-It can be used as an alternative to {cass-migrator} ({cass-migrator-short}).
+svg::sideloader:astra-migration-toolkit.svg[]
 
-* xref:sideloader:sideloader-overview.adoc[{sstable-sideloader}]: {sstable-sideloader} is a service running in {astra-db} that directly imports data from snapshot backups that you've uploaded to {astra-db} from an existing {cass-reg}, {dse}, or {hcd} cluster.
+++++
+</div>
+++++
 
-////
-* https://github.com/datastax/zdm-proxy-automation[{product-automation}] repo for Ansible-based {product-proxy} automation.
-//{product-automation} 2.3.0, which enables ansible scripts and terraform to work with both Ubuntu and RedHat-family Linux distributions.
+[.[&>h2]:!hidden]
+== {empty}
 
-* https://github.com/datastax/dsbulk-migrator[{dsbulk-migrator}] repo for migration of smaller data quantities.
+[subs="macros,attributes"]
+++++
 
-* https://github.com/datastax/cassandra-data-migrator[{cass-migrator}] repo for migration of larger data quantities and where detailed verifications and reconciliation options are needed.
-////
+<h2 class="discrete !text-h1 !mt-12 !mb-6">Migration tools</h2>
 
-////
-Possible related content:
+<div class="grid gap-6 lg:grid-cols-4">
+  <div class="grid gap-4">
 
-https://docs.datastax.com/en/dse/6.8/tooling/migration-path-dse.html
+    svg:ROOT:icons/astra-db-cloud-backup-restores.svg[role="mx-auto max-w-xs md:mx-0 lg:max-w-none"]
 
-https://docs.datastax.com/en/dse/6.9/tooling/migration-path-dse.html
+    <h3 class="discrete !text-h2 !m-0">{product} ({product-short})</h3>
 
-MC only:
+    <p>Comprised of {product-proxy}, {product-utility}, and {product-automation}, you can continue to run your current application and migrate data from the origin to the target database without any downtime. {product-proxy} helps to manage the activity in transition.</p>
 
-https://docs.datastax.com/en/mission-control/migrate/oss-cass-to-mission-control.html
+    <div class="landing-a">
+        xref:ROOT:introduction.adoc[Get started with {product-short}]
+    </div>
 
-https://docs.datastax.com/en/mission-control/migrate/dse-to-mission-control.html
-////
\ No newline at end of file
+  </div>
+  <div class="grid gap-4">
+
+    svg:ROOT:icons/loading-data-into-astra-db.svg[role="mx-auto max-w-xs md:mx-0 lg:max-w-none"]
+
+    <h3 class="discrete !text-h2 !m-0">{cass-migrator} ({cass-migrator-short})</h3>
```

Comment:
@eric-schneider  - The icons align, but is it possible to get the headings to align? 

![image](https://github.com/user-attachments/assets/dd02249e-f9e9-40c4-85ae-b5db44069352)

I realize this may be a feature of all the landing pages.